channeld/channeld.c

```diff
@@ -510,7 +511,7 @@ static void channel_announcement_negotiate(struct peer *peer)
 		 * announcement to MASTER.
 		 * But we won't do this when restart, because we load announcement
 		 * signatures form DB when we reenable Channeld! */
-		if(peer->reconnected == false){
+		if(peer->remote_ann_stored == false){
```

Comment:
See above, just always send the signatures to the master.

Reply:
Yes, "Do The Simplest Thing"